src/NServiceBus.RabbitMQ/RabbitMQTransportInfrastructure.cs

```diff
@@ -25,13 +25,24 @@ public RabbitMQTransportInfrastructure(SettingsHolder settings, string connectio
             var connectionConfiguration = new ConnectionStringParser(settings).Parse(connectionString);
             connectionFactory = new ConnectionFactory(connectionConfiguration);
 
+            SetUsePublisherConfirmsIfPresent(settings, connectionConfiguration);
+
             CreateTopology();
 
             channelProvider = new ChannelProvider(connectionFactory, routingTopology, connectionConfiguration.UsePublisherConfirms);
 
             RequireOutboxConsent = false;
         }
 
+        static void SetUsePublisherConfirmsIfPresent(SettingsHolder settings, ConnectionConfiguration connectionConfiguration)
+        {
+            if (settings.HasSetting(SettingsKeys.UsePublisherConfirms))
+            {
+                var publisherCofirms = settings.Get<bool>(SettingsKeys.UsePublisherConfirms);
+                connectionConfiguration.UsePublisherConfirms = publisherCofirms;
+            }
```

Comment:
It would be nice if we didn't have to do all this and could do something like:

```C#
connectionConfiguration.UsePublisherConfirms = settings.Get<bool>(SettingsKeys.UsePublisherConfirms, false)
```

I.e. if not present, use the second parameter (in this case `false`) as the default. But I guess that would require a change in core?